### PR TITLE
feat(performance-issues): adds a route to redirect to the newest performance issue

### PR DESCRIPTION
**Pull request by scefali**
This PR adds a route to redirect to the latest performance issue that user has access to for a given organization. The purpose of this is so we can use this route in Pendo as it's not possible to construct a dynamic URL given an issue ID.

### CODE REVIEW

File: src/sentry/web/frontend/newest_performance_issue.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        projects = get_projects(request, organization)
+        project_ids = [project.id for project in projects]
+
+        group = (
+            Group.objects.filter(
+                project_id__in=project_ids,
+                status=GroupStatus.UNRESOLVED,
+                # performance issue range
+                type__gte=1000,
+                type__lt=2000,
```
Comment (wedamija) [inline src/sentry/web/frontend/newest_performance_issue.py:33]: Is this just temporary? Would prefer to not have these hardcoded if it's long term

**Review by wedamija — APPROVED** [review]

File: src/sentry/web/frontend/newest_performance_issue.py (hunk 1 above)
Comment (scefali) [reply src/sentry/web/frontend/newest_performance_issue.py:33]: @wedamija Well, I can use the enum for the lower end but there's nothing on the higher end. What would you recommend?

Comment (wedamija) [reply src/sentry/web/frontend/newest_performance_issue.py:33]: We'll probably provide some constants at some point. But if this is short term then we don't need to worry about updating it

Comment (scefali) [reply src/sentry/web/frontend/newest_performance_issue.py:33]: @wedamija If this is still around then, just ping me and I'll update it here.

**Pull request merged**